storageClient/api.go

```diff
@@ -73,12 +73,31 @@ type AddingCapabilityRequest struct {
 	AccountID      uuid.UUID `json:"account_id"`
 }
 
-// AddingClientToGroupRequest wraps values used to add a client to a Group.
-type AddingClientToGroupRequest struct {
-	ClientID       uuid.UUID `json:"client_id"`
-	Name           string    `json:"group_name"`
-	CapabilityName string    `json:"capability_name"`
-	AccountID      uuid.UUID `json:"account_id"`
+// NewMember wraps values needed to add a client to a Group.
+type GroupMember struct {
+	ClientID       uuid.UUID `json:"client_ids"`
+	MembershipKey  string    `json:"membership_key"`
+	CapabilityName []string  `json:"capability_names"`
+}
+
+// AddToGroupRequest wraps the information of all members being added
+type AddToGroupRequest struct {
+	GroupID      uuid.UUID `json:"group_id"`
+	GroupMembers []GroupMember
+}
+
+// AddToGroupResponse wraps values for clients added to a Group.
+type AddToGroupResponse struct {
+	GroupID      uuid.UUID `json:"group_id"`
```

Comment:
Because the client can't add members to a group without knowing the GroupID, I don't think we need to return the GroupID to them in this response. For AddGroupMembers and RemoveGroupMembers the response can be a simple slice of GroupMembers